homeassistant/components/zwave_js/update.py

```diff
@@ -163,7 +164,7 @@ async def async_install(
                     self.node, file
                 )
         except BaseZwaveJSServerError as err:
-            raise err
+            raise HomeAssistantError from err
```

Comment:
Maybe include a message as exception argument?

Reply:
I can just wrap the error in it right to get that text back?

Reply:
Yes, if the original exception has arguments stored they will be used as string representation.